lib/mastodon/media_cli.rb

```diff
@@ -184,6 +184,20 @@ def remove_orphans
       say("Removed #{removed} orphans (approx. #{number_to_human_size(reclaimed_bytes)})#{dry_run}", :green, true)
     end
 
+    desc 'mark-media-missing', 'Clears references to remote media which has been stored locally'
+    long_desc <<~DESC
+      Clears references to file and thumbnails stored locally that originated remotely.
+
+      This is useful if you are restoring a backup without the cache and need Mastodon to fetch remote media again.
+    DESC
+    def mark_media_missing
+      MediaAttachment.cached.where.not(remote_url: '').each do |attachment| 
```

Comment:
```suggestion
      MediaAttachment.cached.where.not(remote_url: '').each do |attachment|
```

Reply:
That's what I get for doing the suggestion on GitHub and not in my usual editor with the usual bells and whistles 🙈

Reply:
I mean, this kind of real-time patching is pretty cool and thank you for the tour of the github `Commit suggestion` interface. 😄